idunn/utils/categories.yml

```
@@ -1,12 +1,22 @@
 categories:
-    # NOTE: This is a legacy alias of `cinema`.
-    #       It may be removed once erdapfel don't rely on it anymore.
+    # NOTE: This currently contains the union of POIs from cinema, theatre and
+    #       attraction (minus artworks and subclass:attraction).
     leisure:
-        pj_what: "cinéma"
-        pj_filters:
-          - "salles de cinéma"
         raw_filters:
           - class: "cinema"
+          - class: "theatre"
+          - subclass: "nightclub"
+          - subclass: "escape_game"
+          - subclass: "golf_course"
+          - subclass: "miniature_golf"
+          - class: "aquarium"
+            subclass: "aquarium"
+          - subclass: "art_gallery"
+          - subclass: "art_center"
+          - subclass: "gallery"
```

`art_gallery` does not seem to be a valid subclass. And `art_center` is actually `arts_centre`.  
(Yes this nomenclature is a bit of a mess).
```suggestion
          - subclass: "arts_center"
          - subclass: "gallery"
```